connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -191,14 +191,31 @@
 
     private MinecraftProtocol protocol;
 
+    private boolean reducedDebugInfo = false;
+
+    @Setter
+    private CustomFormWindow settingsForm;
+
+    // These are used in adventure settings
+    @Setter
+    private int opPermissionLevel = 0;
```

Comment:
I know it's redundant, but with the plugin API incoming having a full Javadoc comment for each of these might be worth it, especially as things like `worldImmutable` may not be clear for some developers.